### PR TITLE
Adding the Lottie Animation in places where loading is there

### PR DESCRIPTION
**Pull request by amanjeetsingh150**
This PR implements Lottie in the project. The PR was worked by @imanserag and me. The json file for the Lottie is in this issue: [ISSUE](https://github.com/Droidcon-Boston/conference-app-android/issues/141).

This implements loading in following places of app:
1. Loading of speaker images in Schedule Adapter.
2. Loading of speaker images in Agenda Details.

Do tell if it needs any change or any other places according to you can be supported with this progress.

Demo:

### CODE REVIEW

**Comment by kenyee** [comment]
looks weird to have an animated loading icon for avatars...UX is ok w/ it?

**Comment by amanjeetsingh150** [comment]
Hey @kenyee I was also confused that's why decide to go forward to make PR and have suggestions from all of you :]

File: Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/ScheduleAdapterItem.kt [hunk 1]
```diff
@@ -66,17 +73,17 @@ class ScheduleAdapterItem internal constructor(
...
     ): ViewHolder {
         return ScheduleAdapterItem.ViewHolder(view, adapter)
     }
 
     override fun bindViewHolder(
-        adapter: FlexibleAdapter<IFlexible<RecyclerView.ViewHolder>>?,
-        holder: ViewHolder,
-        position: Int,
-        payloads: MutableList<Any>
+            adapter: FlexibleAdapter<IFlexible<RecyclerView.ViewHolder>>?,
```
Comment (kenyee) [inline Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/ScheduleAdapterItem.kt]: does this new spacing cause ktlint to flag them?

File: Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/ScheduleAdapterItem.kt [hunk 2]
```diff
@@ -116,9 +123,15 @@ class ScheduleAdapterItem internal constructor(
 
             val context = holder.title.context
 
+
             Glide.with(context).load(itemData.photoUrlMap[itemData.primarySpeakerName])
-                .transform(CircleTransform(context)).placeholder(R.drawable.emo_im_cool).crossFade()
-                .into(holder.avatar)
+                    .asBitmap()
```
Comment (kenyee) [inline Droidcon-Boston/app/src/main/java/com/mentalmachines/droidcon_boston/views/agenda/ScheduleAdapterItem.kt]: why switch to asBitmap?

**Comment by imanserag** [comment]
You have UI | UX approval!

**Comment by AdamMc331** [comment]
I think it looks good, but why did we remove the code styles XML file?

**Comment by amanjeetsingh150** [comment]
Hey @AdamMc331 I have restored the code style file. It must have got deleted by me mistakenly.

Please review now 👍

**Review by AdamMc331 — APPROVED** [review]

**Pull request merged**